docs/get-started/installation/prepare/rpi-setup.md

```diff
@@ -59,92 +63,112 @@ To install Raspberry Pi OS (formerly called Raspbian) on a microSD card from whi
 
     You should be brought back to the initial launch screen.
 
-3.  To make your Raspberry Pi easier to access in the next step, configure your Raspberry Pi's hostname, ssh credentials, and wifi.
-    Click the gear-shaped settings icon in the lower right to bring up the Advanced options menu.
+1.  Now, pick your storage medium.
+    Click `CHOOSE STORAGE`:
+
+    {{< imgproc alt="Raspberry Pi Imager window showing the main page, and the 'Choose Storage' button is selected." src="/get-started/installation/rpi-setup/imager-selected-os.png" resize="800x" declaredimensions=true >}}
+
+    You may have many devices listed, select the microSD card you intend to use in your Raspberry Pi.
+    If this page is blank and you do not have any listed, make sure your microSD card is connected to your computer correctly:
+
+    {{< imgproc alt="The storage screen is shown with a generic SD card available as an option." src="/get-started/installation/rpi-setup/imager-select-storage.png" resize="800x" declaredimensions=true >}}
+
+1.  Now, configure your Raspberry Pi for remote access.
+    Click **Next**.
+    When prompted to apply OS customization settings, select **EDIT SETTINGS**.
 
     {{< imgproc alt="Raspberry Pi Imager window showing gear-shaped settings icon is selected." src="/get-started/installation/rpi-setup/advanced-options.png" resize="800x" declaredimensions=true >}}
 
     {{% alert title="Important" color="note" %}}
 
-If you are using a non-Raspberry Pi OS, altering the Advanced options will cause the initial boot to fail.
+    If you are using a non-Raspberry Pi OS, altering the OS customization settings will cause the initial boot to fail.
 
     {{% /alert %}}
 
-    {{< imgproc alt="Raspberry Pi Imager window showing the advanced options menu." src="/get-started/installation/rpi-setup/imager-set-hostname.png" resize="800x" declaredimensions=true >}}
+    {{< imgproc alt="Raspberry Pi Imager window showing the advanced options menu." src="/get-started/installation/rpi-setup/imager-set-hostname.png" resize="400x" declaredimensions=true >}}
 
-    Check `Set hostname` and enter the name you would like to access the Pi by in that field.
+    Check **Set hostname** and enter the name you would like to access the Pi by in that field.
 
     There are two ways you can secure your Raspberry Pi: with an SSH key or with password authentication.
-
-    To use the SSH key method: check `Enable SSH`.
     Using SSH Keys for authentication is a great way of securing your Raspberry Pi as only someone with the private SSH key will be able to authenticate to your system.
-    If you select `Allow public-key authentication only`, and the section `set authorized_ keys for 'pi'` is pre-populated, that means you do have an existing public SSH key that is ready to use.
-    In that case, you do not have to change this section.
 
+    To set up SSH authentication:
+
+    1.  Click on **SERVICES**.
+    1.  Check **Enable SSH**.
+
+    1.  {{< tabs >}}
```

Comment:
Why is this a list item? See above for why this is rendering weirdly.

Reply:
It is a list item because it is the third step(s) to set up ssh authentication